spec/models/student_spec.rb

```diff
@@ -28,4 +29,34 @@
     end
 
   end
+
+  describe 'when erasing a student from the student_queue' do
+    before :each do
+      student_info = {:first_name => 'Athina',
+                      :sid => 11234832}
+      @student = FactoryGirl.build(:student, student_info)
+    end
+    it 'checks if the student has a entry in the student queue' do
+      expect(@student).to receive(:student_queue)
+      @student.destroy_student_queue
```

Comment:
Why do you have this line here? Does it actually destroy the student in the test?